src/content/en/updates/2018/03/clipboardapi.md

```diff
@@ -0,0 +1,269 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Copycats rejoice! Async Clipboard API is unblocking copy & paste in Chrome 66.
+
+{# wf_updated_on: 2018-03-13 #}
+{# wf_published_on: 2018-03-13 #}
+{# wf_tags: cutandcopy,execcommand,input,chrome66 #}
+{# wf_featured_image: /web/updates/images/generic/share.png #}
+{# wf_featured_snippet: Async Clipboard API simplifies permissions-friendly copy & paste. #}
+{# wf_blink_components: Blink>Input #}
```

Comment:
Let's use `Blink>DataTransfer`, same as the chromestatus entry.

Reply:
Perfect, will do.